source/class.cpp

```diff
@@ -160,7 +160,7 @@ string bind_data_member(FieldDecl const *d, string const &class_qualified_name_)
 	}
 
 	if( d->getType().isConstQualified() ) return ".def_readonly(\"{}\", &{}::{})"_format(d->getNameAsString(), class_qualified_name, d->getNameAsString());
-	else return ".def_readwrite(\"{}\", &{}::{})"_format(d->getNameAsString(), class_qualified_name, d->getNameAsString());
+	return ".def_readwrite(\"{}\", &{}::{})"_format(d->getNameAsString(), class_qualified_name, d->getNameAsString());
```

Comment:
@jwuttke could you please argue why the proposed style is better compare to original layout? Thanks,

Reply:
LLVM style guide: https://llvm.org/docs/CodingStandards.html#don-t-use-else-after-a-return.

However, quick web search shows that there is also some opposition to this rule.

Up to you to decide. I can of course reset this PR to the second but last commit.

Reply:
Hm... i am not sure about this. Also, it looks like this PR is trying to use this guide line: https://llvm.org/docs/CodingStandards.html#use-early-exits-and-continue-to-simplify-code which i also find depending on particular coding taste.